.github/workflows/ci.yml

```diff
@@ -125,3 +126,25 @@ jobs:
 
       - name: Elm Tests
         run: npm run elm-test --prefix assets
+
+      - name: Xref Graph Gen/Upload
+        id: xref
+        run: |
+          mix xref graph --format dot
+          sudo apt-get install -y jq graphviz
```

Comment:
It probably doesn't matter, but I wonder if this should be up with the other installation bits.

Reply:
Not sure if it is like docker and could realize that it hasn't changed in that layer (and then not execute it again).

Reply:
I was trying to get it to fail as fast as possible, so deferring installation steps until needed. Though it is probably a bunch cleaner if I do it in one spot.

Reply:
Will refactor in the next PR, will still be in this file.